packages/integrations/image/src/index.ts

```diff
@@ -88,8 +94,15 @@ export default function integration(options: IntegrationOptions = {}): AstroInte
 					defaultLoader,
 				};
 			},
-			'astro:build:start': async ({ buildConfig }) => {
-				_buildConfig = buildConfig;
+			'astro:config:done': ({ config }) => {
+				_config = config;
+				_buildConfig = config.build;
+			},
+			'astro:build:start': ({ buildConfig }) => {
+				// Backwards compat
+				if(needsBuildConfig) {
+					_buildConfig = buildConfig;
+				}
```

Comment:
So, this line exists for backwards compatibility. If you are using Astro < 1.5.0 then this line will get `_buildConfig` from the `'astro:build:start'` hook. If you are using Astro >= 1.5.0 then it will get it from the `'astro:config:done'` hook here: https://github.com/withastro/astro/pull/5056/files#diff-0000f87084d8080bf428d6b3b7dc84cda4f01465d9a3e95c75bbc38b8129e1e5R99

In the future when we do a new major (or minor in the case of astro/image) we can get rid of this code and only support users of Astro >= 1.5.0